backend/model/Notifications.js

```diff
@@ -11,37 +11,17 @@ const users = db.collection("users");
 // Firestore
 const NotificationsAPI = {
     getNotifications: async function (uid) {
-        // UNPAGINATED VERSION (this is more preferable for frontend)
-        // var notificationCollection = usersCollection
-        //   .doc(uid)
-        //   .collection("notifications")
-        //   .orderBy("date_time")
-        // const notificationSnapshot = await notificationCollection.get();
-        // let notifications = [];
-        // notificationSnapshot.forEach(doc => {
-        //   notifications = [...notifications, doc.data()];
-        // });
-        // return notifications
-        // PAGINATED VERSION
-        var first = db
-            .doc(currUser.uid)
-            .collection("notifications")
-            .orderBy("date_time")
-            .limit(5);
-        return first.get().then(function (documentSnapshots) {
-            // Get the last visible document
-            var lastVisible =
-                documentSnapshots.docs[documentSnapshots.docs.length - 1];
-            console.log("last", lastVisible);
-
-            // Construct a new query starting at this document,
-            // get the next 25 cities.
-            var next = db
-                .doc(currUser.uid)
-                .collection("notifications")
-                .orderBy("date_time")
-                .limit(5);
+        // UNPAGINATED
+        var notificationCollection = users
+          .doc(uid)
+          .collection("notifications")
+          .orderBy("date_time")
+        const notificationSnapshot = await notificationCollection.get();
+        let notifications = [];
+        notificationSnapshot.forEach(doc => {
+          notifications = [...notifications, doc.data()];
```

Comment:
Notifications should be handling the logic, so the route is just calling this and passing back the return